peps/pep-0733.rst

```diff
@@ -0,0 +1,634 @@
+PEP: 733
+Title: An Evaluation of Python's Public C API
+Author: Erlend E. Aasland <erlend@python.org>, Domenico Andreoli <>, Stefan Behnel <stefan_ml@behnel.de>, Carl Friedrich Bolz-Tereick <cfbolz@gmx.de>, Simon Cross <>, Steve Dower <steve.dower@python.org>, Tim Felgentreff <tim.felgentreff@oracle.com>, David Hewitt <>, Shantanu Jain <hauntsaninja at gmail.com>, Wenzel Jakob <>, Irit Katriel <irit@python.org>, Marc-Andre Lemburg <mal@lemburg.com>, Dong-hee Na <>, Karl Nelson <>, Ronald Oussoren <ronaldoussoren@mac.com>, Matti Picus <>, Antoine Pitrou <solipsis@pitrou.net>, Benedikt Reinartz <>, Guido van Rossum <guido@python.org>, Neil Schemenauer <nas@arctrix.com>, Mark Shannon <mark@hotpy.org>, Stepan Sindelar <stepan.sindelar@oracle.com>, Gregory P. Smith <greg@krypto.org>, Eric Snow <ericsnowcurrently@gmail.com>, Victor Stinner <vstinner@python.org>, Serhiy Storchaka <>, Petr Viktorin <encukou@gmail.com>, Carol Willing <willingc@gmail.com>, William Woodruff <william@yossarian.net>, David Woods <>, Jelle Zijlstra <jelle.zijlstra@gmail.com>
```

Comment:
I'm not sure that all people who contributed to https://github.com/capi-workgroup/problems/#problems-in-the-python-c-api project expected to get their name and email address listed here. The project README says nothing about a PEP.

Listing all people is great. I just suggest to move the list outside the Authors field. And maybe without email addresses, but maybe with a link to their GitHub profile, since GitHub was used? I don't know.

Reply:
I only put emails that are already on other PEPs (so already public) or were specifically given for this purpose.